mw-webapp/src/component/header/Header.tsx

```diff
@@ -147,6 +148,7 @@ export const Header = (props: HeaderProps) => {
       <Link
         className={styles.logo}
         path={pages.allWays.getPath({})}
+        dataCy={headerAccessIds.items.allWaysLogoBtn}
```

Comment:
Also logo in the header does it is not a button, it is a link, but it does not matter to be honest, so: 
`headerAccessIds.items.allWaysLogoBtn` -> `headerAccessIds.items.logo`

Reply:
If I changed ID to 'logo', then in this case I also leave only 'logo' for the selector, right?
Id: logo
Selector: getLogo

Reply:
not clear what does this lines mean 
```
Id: logo
Selector: getLogo
```

Reply:
yeah, good